src/main/java/org/folio/circulation/domain/RequestStatus.java

```diff
@@ -2,31 +2,38 @@
 
 import static org.apache.commons.lang3.StringUtils.equalsIgnoreCase;
 
-import java.util.ArrayList;
 import java.util.Arrays;
-
-import org.folio.circulation.domain.representations.RequestProperties;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.stream.Collectors;
 
 import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+
+import org.folio.circulation.domain.representations.RequestProperties;
 
 public enum RequestStatus {
   NONE(""),
   OPEN_NOT_YET_FILLED("Open - Not yet filled"),
   OPEN_AWAITING_PICKUP("Open - Awaiting pickup"),
   OPEN_IN_TRANSIT("Open - In transit"),
+  OPEN_AWAITING_DELIVERY("Open - Awaiting delivery"),
   CLOSED_FILLED("Closed - Filled"),
   CLOSED_CANCELLED("Closed - Cancelled"),
   CLOSED_UNFILLED("Closed - Unfilled"),
   CLOSED_PICKUP_EXPIRED("Closed - Pickup expired");
 
+  private static final EnumSet<RequestStatus> VALID_STATUSES =
+    EnumSet.range(OPEN_NOT_YET_FILLED, CLOSED_PICKUP_EXPIRED);
+  private static final EnumSet<RequestStatus> OPEN_STATUSES =
+    EnumSet.range(OPEN_NOT_YET_FILLED, OPEN_AWAITING_DELIVERY);
```

Comment:
I think if we are going to use this kind of succinct definition, I think we need better checks to ensure that the definition is correct when a new status is added or the list is rearranged.